fastcrypto-tbls/src/types.rs

```diff
@@ -50,6 +50,41 @@ pub struct IndexedValue<A> {
     pub value: A,
 }
 
+/// Basic wrapper of a set of values that are not associated with indexes, assuming the indexes are known to all
+/// parties. Used to reduce the size of the messages in the protocol.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct UnindexedValues<A> {
+    pub values: Vec<A>,
+}
+
+impl<A> From<Vec<IndexedValue<A>>> for UnindexedValues<A> {
+    fn from(index_values: Vec<IndexedValue<A>>) -> Self {
+        let mut values: Vec<A> = Vec::new();
```

Comment:
nit-maybe use `Vec::with_capacity` or a functional-style init to avoid reallocs?